charts/loki-stack/templates/datasources.yaml

```diff
@@ -21,6 +21,8 @@ data:
       url: http://{{(include "loki.serviceName" .)}}:{{ .Values.loki.service.port }}
       version: 1
       isDefault: {{ .Values.loki.isDefault }}
+      jsonData:
+        maxLines: {{ .Values.grafana.sidecar.datasources.enabled }}
```

Comment:
Shouldn't be:
{{ .Values.grafana.sidecar.datasources.maxLines }}

Reply:
totally, my bad. Migrated this from another PR and copy pasted